contracts/SSVNetwork.sol

```diff
@@ -168,7 +168,17 @@ contract SSVNetwork is
     function registerValidator(
         bytes calldata publicKey,
         uint64[] memory operatorIds,
-        bytes calldata sharesData,
+        bytes calldata shares,
```

Comment:
Minor inconsistency (shares vs sharesData) in the parameter naming for `registerValidators()` between the **SSVClusters.sol** and **SSVNetwork.sol** contract. 
While this is not a critical issue, aligning the parameter names would enhance code readability and maintain consistency across our codebase.